root/artist/header.tt

```diff
@@ -1,10 +1,10 @@
-<div class="artistheader">
-    <h1>[% link_entity(artist) %]</h1>
+[% header = artist.type_name _ 'icon' %]
+<div class = "artistheader [% header.lower() %]">
+  <h1>[% link_entity(artist) %]</h1>
```

Comment:
The indentation of this line was changed. :) Add back the two spaces it had (four total).